apps/account-api/test/accounts.controller.e2e-spec.ts

```diff
@@ -95,4 +97,38 @@ describe('Account Controller', () => {
       .expect((res) => res.body.msaId === validMsaId)
       .expect((res) => res.body.handle.base_handle === handle);
   });
+
+  it('(GET) /v1/accounts/retireMsa/:accountId get payload for retireMsa, given a valid accountId', async () => {
+    const accountId = users[0].keypair.address;
+    await request(app.getHttpServer())
+      .get(`/v1/accounts/retireMsa/${accountId}`)
+      .expect(200)
+      .expect((res) => res.body.unsignedPayload === '')
+      .expect((res) => res.body.encodedPayload === '');
+  });
+
+  // if (
+  //   ('(GET) /v1/accounts/retireMsa/:accountId get payload for retireMsa, given an invalid accountId', async () => {})
+  // );
+
+  it.only('(POST) /v1/accounts/retireMsa post retireMsa', async () => {
```

Comment:
Remove `.only`

Reply:
If there are other tests in this file that are broken, we should either (a) fix them, or (b) mark the broken ones with `.skip`, but let's not commit `.only`